WMF Framework/FakeProgressController.swift

```diff
@@ -62,10 +64,10 @@ public class FakeProgressController: NSObject {
             if isProgressHidden {
                 NSObject.cancelPreviousPerformRequests(withTarget: self, selector: #selector(showProgress), object: nil)
                 NSObject.cancelPreviousPerformRequests(withTarget: self, selector: #selector(incrementProgress), object: nil)
-                perform(#selector(hideProgress), with: nil, afterDelay: 0.7)
+                perform(#selector(hideProgress), with: nil, afterDelay: minVisibleDuration)
             } else {
                 NSObject.cancelPreviousPerformRequests(withTarget: self, selector: #selector(hideProgress), object: nil)
-                showProgress()
+                perform(#selector(showProgress), with: self, afterDelay: delay)
```

Comment:
This is missing a cancel of previous `showProgress` requests before kicking off the new one - as implemented `showProgress` calls would be made for every keystroke